pkg/config/config_test.go

```diff
@@ -257,6 +257,7 @@ image_copy_tmp_dir="storage"`
 			gomega.Expect(defaultConfig.Containers.Env).To(gomega.BeEquivalentTo(envs))
 			gomega.Expect(defaultConfig.Containers.PidsLimit).To(gomega.BeEquivalentTo(2048))
 			gomega.Expect(defaultConfig.Network.CNIPluginDirs).To(gomega.Equal(pluginDirs))
+			gomega.Expect(defaultConfig.Network.NetavarkPluginDirs).To(gomega.Equal([]string{"/usr/netavark"}))
```

Comment:
Not a blocker: Should the default be `/usr/netavark` or `/usr/podman/netavark`. I mean atleast something which tells that following configuration or plugin is a subset of podman.

Reply:
or maybe `$runroot/netavark/`

Reply:
This is just the test config, the default paths are documented as 
"/usr/local/libexec/netavark",
	"/usr/libexec/netavark",
	"/usr/local/lib/netavark",
	"/usr/lib/netavark",

Reply:
ah fair enough.